calour/tests/test_analysis.py

```diff
@@ -48,40 +48,52 @@ def test_diff_abundance(self):
         for cid in expected_ids:
             print(self.test1.feature_metadata.index[cid])
             self.assertIn(self.test1.feature_metadata.index[cid], dd.feature_metadata.index)
+
+    def test_correlation(self):
```

Comment:
anyway, they are not good tests because no matter what you change conditions, the result are the same. essentially it just tests one condition.